templates/block-templates/single-no-access.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:template-part {"slug":"header"} /-->
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:llms/php-template {"template":"single-no-access", "title":"LifterLMS Single Template requiring Membership"} /--></div>
```

Comment:
```suggestion
<div class="wp-block-group"><!-- wp:llms/php-template {"template":"single-no-access", "title":"LifterLMS Template Access Restricted"} /--></div>
```

This is not often displayed but I don't think it's displayed exclusively when membership is required so I think the title should reflect that to prevent confusion or bug reports or whatever.

Reply:
mmm I think you're right :D